src/action.cpp

```diff
@@ -1019,47 +1014,57 @@ cata::optional<tripoint> choose_direction( const std::string &message, const boo
     return cata::nullopt;
 }
 
-cata::optional<tripoint> choose_adjacent( const std::string &message )
-{
-    return choose_adjacent( message, false );
-}
-
-cata::optional<tripoint> choose_adjacent( const std::string &message, const bool allow_vertical )
+cata::optional<tripoint> choose_adjacent( const std::string &message, bool allow_vertical )
 {
     const cata::optional<tripoint> dir = choose_direction( message, allow_vertical );
     return dir ? *dir + g->u.pos() : dir;
 }
 
 cata::optional<tripoint> choose_adjacent_highlight( const std::string &message,
-        const action_id action_to_highlight )
-{
-    return choose_adjacent_highlight( message, action_to_highlight, false );
-}
-
-cata::optional<tripoint> choose_adjacent_highlight( const std::string &message,
-        const action_id action_to_highlight, const bool allow_vertical )
+        const action_id action, bool allow_vertical )
```

Comment:
```suggestion
        const action_id action, const bool allow_vertical )
```

Reply:
What about clang-tidy - should we really use const for simple types like bool or int?

Reply:
> should we really use const for simple types like bool or int?

Yes. Why not?

Reply:
> What about clang-tidy

`clang-tidy` only checks if const is used in the declaration of functions, where constness of primitive types does not make any difference. In the definition of functions constness is desired if it is not supposed to be modified.